runtime/ftplugin/mom.vim

```diff
@@ -0,0 +1,13 @@
+" Vim filetype plugin
+" Language:	roff(7)
+" Maintainer:	Aman Verma
+" Homepage:	https://github.com/a-vrma/vim-nroff-ftplugin
+" Previous Maintainer: Chris Spiegel <cspiegel@gmail.com>
+"		2024 May 24 by Riley Bruins <ribru17@gmail.com> ('commentstring')
```

Comment:
This does not seem to make sense, given that this is a new filetype plugin.

Reply:
Removed